app/src/main/java/edu/uc/mille5bm/smartnutrition/MainActivity.kt

```diff
@@ -9,8 +9,8 @@ class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
-        findViewById<View>(R.id.button1).setOnClickListener { startActivity(Intent(applicationContext, NutritionInfoActivity::class.java)) }
-        findViewById<View>(R.id.button2).setOnClickListener { startActivity(Intent(applicationContext, CalorieCounterActivity::class.java)) }
-        findViewById<View>(R.id.button3).setOnClickListener { startActivity(Intent(applicationContext, HealthTrackerActivity::class.java)) }
+        findViewById<View>(R.id.btnNutritionInfo).setOnClickListener { startActivity(Intent(applicationContext, NutritionInfoActivity::class.java)) }
+        findViewById<View>(R.id.btnCalorieCounter).setOnClickListener { startActivity(Intent(applicationContext, CalorieCounterActivity::class.java)) }
+        findViewById<View>(R.id.btnHealthTracker).setOnClickListener { startActivity(Intent(applicationContext, HealthTrackerActivity::class.java)) }
```

Comment:
Good suggestion, though this duplicates a previous code review.  When that occurs, first in wins.